### ISSUE TITLE
Problem with Roccat Tyon

### ISSUE THREAD

**Issue by Serraniel** [CosminLazar/roccat-talk-csharp#1]
Hey,

I tryed your work with your testproject. The console launched and was running without errors until the end. I also debugged and there were nothing to see that might cause issues. 
But I could not see any result in the color of my mouse. I own a Roccat Tyon which supports TalkFX. 

Do you have any idea, why nothing happend?

**Comment by CosminLazar** [comment]
Hi Serraniel, 
In theory you should be able to control your Roccat Tyon. Make sure you have the TalkFX software running (http://www.roccat.org/dk-en/Support/Downloads/#/22/Gaming-Software/49/ROCCAT----Talk---FX/) then I suggest you play a bit with the parameters in the `TalkFxTEst` method.

I suggest you start by changing the colour of your mouse to something non white
`connection.SetLedRgb(Zone.Event, KeyEffect.On, Speed.Normal, new Color(255, 255, 255));`

and if it doesn't work try the following:
- `connection.SetLedRgb(Zone.Event, KeyEffect.On, Speed. NoChange, new Color(255, 255, 255));`
- `connection.SetLedRgb(Zone.Ambient, KeyEffect.On, Speed. Normal, new Color(255, 255, 255));`
- basically just try different enum value combinations

Let me know if it works afterwards.

**Comment by Serraniel** [comment]
Hy,
thanks for your reply. I already tried that, without any success. Also reinstalled Talk-FX twice. maybe I will try to contact the Roccat support next week and ask if they have any idea why it does not work correctly. (The taskmanager shows me that Roccat Talk is running.)

**Comment by CosminLazar** [comment]
Too bad it doesn't work for you, however, event if it would work, I don't know how you'd be able to set the RGB to each individual location of your mouse using the current API (from what I have seen you can customise the scrolling wheel colour, and the bottom colour of your mouse).

If you do go forward and contact them, please also let me know of their answer.
